DAS/pyproject.toml

```diff
@@ -5,13 +5,14 @@ description = ""
 authors = ["Blake <bhay0010@student.monash.edu>"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.9"
```

Comment:
Just to confirm, will the repo not work with python 3.7 going forward?

Reply:
The `DAS` folder and all the files contained in it will not work with python 3.7 anymore. I was running into issues with python 3.7 with `mqtt-logger`. Is there any reason that we need to stay on python 3.7?

Reply:
it's just a pin running` poetry install` on the Pi 😅, if we change python version we would need to run poetry install again in the new virtual env. But it should be alright, as long as we don't pull these changes onto the raspberry pi.

Reply:
Ah that makes sense. I'll try and decrease the python version requirements.